sfcasts/ep1/data-providers.md

```diff
@@ -1,42 +1,42 @@
 # Data Providers
 
-We treat our source code as first-class citizens. Why not do the same with our
+We treat our source code as a first-class citizen. That means, among other things,
+we avoid duplication. Why not do the same with our
 tests? Our three tests for the size are... repetitive. They test the same thing
-just with slightly different input and then a different assertion. We can improve
-this by using PHPUnit's Data Providers - which run the same test over and over
-with different sets of data.
+just with *slightly* different input and then a different assertion. Is there
+a way to improve this? Absolutely: thanks to PHPUnit Data Providers.
```

Comment:
just a little showman ship here. Get them excited about this phpunit data providers feature. But, no need to explain it yet. Show things first, explain after.